duo-client/src/test/java/com/duosecurity/client/HttpRateLimitRetryTest.java

```diff
@@ -22,8 +22,6 @@
 public class HttpRateLimitRetryTest {
     @Mock
     private OkHttpClient httpClient;
-    @Mock
-    private Random random;
```

Comment:
```
org.mockito.exceptions.base.MockitoException: 
Mockito cannot mock this class: class java.util.Random.
```
This was the problem, and quick google search lead me to know java 17 don't allow mock to tap random method anymore. The next easiest thing I can think of is to create a helper method so we can just mock the helper method.

Reply:
I think this is fine, https://security.snyk.io/package/maven/com.squareup.okhttp3:okhttp/4.10.0 says there are no vulnerabilities and it was released june of last year. They're already working on 5.0.0-alpha versions, so I think 4.10.0 is safe.

Reply:
lgtm!